_posts/2014-01-19-breaking-change-formatter.md

```diff
@@ -3,10 +3,9 @@ title: Breaking change to ESLint default formatter
 tags:
   - breaking change
   - formatter
+blogDate: 19 January 2014
```

Comment:
We don’t need this field because the date is in the filename. From [the Eleventy docs](https://www.11ty.dev/docs/dates/):

> If a date key is omitted from the file, we then look for a YYYY-MM-DD format anywhere in the file path (even folders). If there are multiple dates found, the first is used.

Reply:
I can remove the blogDate fields from the posts.